client/src/www/app/outline_server_repository/server.ts

```diff
@@ -80,28 +80,24 @@ export class OutlineServer implements Server {
   }
 
   get address() {
```

Comment:
Done for name anc config location. I had to keep it for address in order to keep it read-only. I can't use `readonly` because I need to assign to it internally.